guides/common/modules/proc_installing-satellite-ansible-modules-via-rpm.adoc

```diff
@@ -0,0 +1,35 @@
+[id="installing-satellite-ansible-modules-via-rpm_{context}"]
+== Installing the {Project} Ansible Modules from RPM
+
+Use this procedure to install the {Project} Ansible modules.
+
+ifeval::["{build}" == "satellite"]
+.Prerequisite
+
+* Ensure that the Ansible 2.9 or later repository is enabled.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable rhel-7-server-ansible-2.9-rpms
+# {foreman-maintain} packages update ansible
+----
+endif::[]
+
+.Procedure
+
+ifeval::["{build}" == "satellite"]
+
+. Install the RPM using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} ansible-collection-redhat-satellite
+----
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+. Install the RPM from the client repository on yum.theforeman.org using the following command:
+----
+# yum install ansible-collection-theforeman-foreman
```

Comment:
```suggestion
# {package-install-project} ansible-collection-theforeman-foreman
```

Reply:
Since evgeni mentioned it's yum install for foreman, hadn't changed this. though attribute makes sense as well as it will replace for foreman accordingly too.